randovania/games/dread/json_data/Artaria.txt

```diff
@@ -559,9 +559,7 @@ Extra - asset_id: collision_camera_005
   > Door to White EMMI Arena
       Trivial
   > Dock to Early Cloak Room (Water Blob)
-      Any of the following:
-          Movement (Intermediate)
-          Movement (Beginner) and Lay Bomb
+      Morph Ball and Movement (Intermediate)
```

Comment:
Would it be worth considering having "Movement (Intermediate)" OR "Movement (Beginner) And Lay Bomb"? Or do bombs not help get into that diagonal tunnel at all?

Reply:
it can help you get in the hole, but im not sure if it would be any easier getting through it. i can try testing it